spec/directions/bake_non_introduction_pages_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe Kitchen::Directions::BakeNonIntroductionPages do
+  let(:chapter) do
+    book_containing(html:
+      one_chapter_with_one_page_containing(
+        <<~HTML
+          <div data-type="metadata" style="display: none;">
+            <div class="description" itemprop="description" data-type="description">
+              <ul>
+                <li>Use functional notation to evaluate a function.</li>
+              </ul>
+          </div>
```

Comment:
this is not a big deal at all but the indentation is a little off for this closing div tag

Reply:
oops, fixed